airflow/models/skipmixin.py

```diff
@@ -140,6 +140,8 @@ def skip_all_except(self, ti: TaskInstance, branch_task_ids: Union[str, Iterable
         self.log.info("Following branch %s", branch_task_ids)
         if isinstance(branch_task_ids, str):
             branch_task_ids = {branch_task_ids}
+        elif branch_task_ids is None:
+            branch_task_ids = {}
```

Comment:
see https://github.com/apache/airflow/issues/18454 and https://github.com/apache/airflow/pull/18471
If we now we are handling None then we need also to change
https://github.com/apache/airflow/blob/9b05f027c3411ff5a5fec71ccf6826207ff9016d/airflow/operators/python.py#L221

Reply:
thanks for the review @eladka